package-lock.json

```diff
@@ -1,14745 +1,8 @@
 {
   "name": "weave-gitops",
   "version": "0.0.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
   "requires": true,
```

Comment:
I would not expect anything to change here. You can probably just checkout this file on `main` to resolve.